include/openthread/ncp.h

```diff
@@ -89,6 +89,16 @@ void otNcpHdlcReceive(const uint8_t *aBuf, uint16_t aBufLength);
  */
 void otNcpHdlcInit(otInstance *aInstance, otNcpHdlcSendCallback aSendCallback);
 
+/**
+ * Initialize the NCP based on HDLC framing.
+ *
+ * @param[in]  aInstances       The OpenThread instance pointers array.
+ * @param[in]  aCount           Number of elements in the array.
+ * @param[in]  aSendCallback    The function pointer used to send NCP data.
+ *
+ */
+void otNcpHdlcInitMulti(otInstance **aInstance, uint8_t aCount, otNcpHdlcSendCallback aSendCallback);
```

Comment:
Should it be wrapped under `#ifdef OPENTHREAD_CONFIG_MULTIPAN_RCP_ENABLE && OPENTHREAD_RADIO` or `OPENTHREAD_CONFIG_MULTIPLE_STATIC_INSTANCE_ENABLE`?